gateway/modules/schema/validatecreate.go

```diff
@@ -168,6 +168,9 @@ func (s *Schema) checkType(ctx context.Context, col string, value interface{}, f
 
 	case map[string]interface{}:
 		if fieldValue.Kind == model.TypeJSON {
+			if dbType, _ := s.crud.GetDBType(dbAlias); model.DBType(dbType) == model.Mongo {
```

Comment:
Extremely dangerous. Can't we have the schema module store dbtype against an alias?